demo/featmap_vis_demo.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Please write here how it is used and tell the user which models we have tested, because we cannot guarantee that all models are working correctly

Reply:
And add: For a detailed explanation of the usage, please refer to mmyolo's documentation

Reply:
> And add: For a detailed explanation of the usage, please refer to mmyolo's documentation

added in the `update usage` commit